proxy/http/remap/RemapConfig.cc

```diff
@@ -949,7 +949,7 @@ remap_parse_config_bti(const char *path, BUILD_TABLE_INFO *bti)
     switch (ec.value()) {
     case ENOENT:
       Warning("Can't open remapping configuration file %s - %s", path, strerror(ec.value()));
-      break;
+      return true;
```

Comment:
Shouldn't that be `false`?

Reply:
I had `false` earlier but, that fails some autest that seems to expect true for a non-existent file.